release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -72,6 +72,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-web-console"]
 === Web console
 
+[id="ocp-4-7-web-console-insights-plugin"]
+==== Insights plug-in
+
+The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#displaying-the-insights-status-in-the-web-console_using-insights-to-identify-issues-with-your-cluster[Insights plug-in] is now integrated into the {product-title} web console. Insights provides cluster health data, such as the number of total issues and total risks of the issues. Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low* and you can quickly navigate to {cloud-redhat-com} for further details about the issues and, if available, how to fix them.
```

Comment:
The last sentence could be broken up and the last part needs to be slightly reworded, perhaps just by removing "if available". Something like:
> Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low*. You can quickly navigate to {cloud-redhat-com} for further details about the found issues and how to fix them.